api/v1beta2/cryostat_types.go

```diff
@@ -587,6 +591,21 @@ type CertificateSecret struct {
 	CertificateKey *string `json:"certificateKey,omitempty"`
 }
 
+type Credential struct {
```

Comment:
Is this used? I think the Operator just needs to know about the secrets to use for mounting volumes, but this is a detail about the contents of the secrets which the Operator should not actually deal in.

Reply:
I think this came back somehow, or maybe was only removed from the `v1beta1` changes before those were also removed?